### PR TITLE
Expose flag for enabling public IPFS node

### PR DESCRIPTION
**Pull request by pyropy**
- Expose flag for enabling public IPFS node
- Add documentation on how to connect devstack to public IPFS node

### CODE REVIEW

File: docs/running_locally.md [hunk 1]
```diff
@@ -31,14 +31,14 @@ go build
 
 This will start a 3 node Bacalhau cluster connected with libp2p.
 
-Each node has its own ipfs server isolated using the `IPFS_PATH` environment variable and its own API RPC server isolated using a random port.
+Each node has its own ipfs server isolated using the `IPFS_PATH` environment variable and its own API RPC server isolated using a random port. These IPFS nodes are not connected to public IPFS network. If you wish to connect devstack to public IPFS network, you can include `--public-ipfs` flag or starting your own IPFS node on same machine as devstack and connecting it by running following command after starting devstack `ipfs swarm connect $BACALHAU_IPFS_SWARM_ADDRESSE` (`BACALHAU_IPFS_SWARM_ADDRESSE` is an environment variable prompted after starting devstack).
```
Comment (simonwo) [inline docs/running_locally.md:34]:
`````suggestion
Each node has its own IPFS server isolated using the `IPFS_PATH` environment variable and its own API RPC server isolated using a random port. These IPFS nodes are not connected to the public IPFS network. If you wish to connect the devstack to the public IPFS network, you can include `--public-ipfs` flag.

You can also use your own IPFS node and connect it to the devstack by running (after starting the devstack):
```
# $BACALHAU_IPFS_SWARM_ADDRESSES is an environment variable given after starting devstack
ipfs swarm connect $BACALHAU_IPFS_SWARM_ADDRESSES
```
`````

File: pkg/devstack/devstack.go [hunk 2]
```diff
@@ -381,6 +383,16 @@ export LOTUS_PATH=%s
...
 	}
 
+	if !stack.PublicIPFSMode {
+		summaryShellVariablesString += `
+
+By default devstack is not running on public IPFS network.
+If you wish to connect devstack to public IPFS network consider running new IPFS node daemon localy
+and then connecting it to bacalhau using command bellow or by adding --public-ipfs flag:
+
+ipfs swarm connect $BACALHAU_IPFS_SWARM_ADDRESSES`
```
Comment (simonwo) [inline pkg/devstack/devstack.go:393]:
```suggestion
By default devstack is not running on the public IPFS network.
If you wish to connect devstack to the public IPFS network add the --public-ipfs flag.
You can also run a new IPFS daemon locally and connect it to Bacalhau using:

ipfs swarm connect $BACALHAU_IPFS_SWARM_ADDRESSES
```

**Review by simonwo — COMMENTED** [review]
Looks great! Just some suggestions on language.

**Comment by pyropy** [comment]
> Looks great! Just some suggestions on language.

Thank you, suggestions have been committed

**Review by simonwo — APPROVED** [review]
Looks like my suggestions had a syntax/linter error, sorry!

**Comment by pyropy** [comment]
> Looks like my suggestions had a syntax/linter error, sorry!

Fixed lint issue

**Pull request merged**